### ISSUE TITLE
TOC is not displayed

### ISSUE THREAD

**Issue by gsmolinski** [r-lib/pkgdown#2215]
I think it is not reported yet: table of contents is not displayed, .e.g here: https://pkgdown.r-lib.org/articles/customise.html.

This problem exists in many places (I have checked my package, `roxygen2` and `pkgdown`), I'm quite sure when I was building the site locally today (for my package), I saw a toc, but I don't see it on GitHub (I don't know if it makes any sense). The JS error is:

> Uncaught ReferenceError: Toc is not defined

in `pkgdown.js:7:5`. Code:

```
(function($) {
  $(function() {

    $('nav.navbar').headroom();

    Toc.init({
      $nav: $("#toc"),
      $scope: $("main h2, main h3, main h4, main h5, main h6")
    });

    if ($('#toc').length) {
      $('body').scrollspy({
        target: '#toc',
        offset: $("nav.navbar").outerHeight() + 1
      });
    }
```

**Comment by maelle** [comment]
Can you try re-installing pkgdown from GitHub (`remotes::install_github("r-lib/pkgdown")`) and re-build the website? I wonder whether it is the same problem as in #2207. Thank you! 

(the fact it appears might be due to the browser cache)

**Comment by gsmolinski** [comment]
I'm not sure if my skills are enough for this :). So - I have used `pkgdown::build_articles()` (after re-installing `pkgdown` from GitHub as you said) and everything was looking good. I couldn't use `pkgdown::build_site()`, because I have some problem with NEWS:

```
Error in curl::curl_fetch_memory(url, handle = handle) : 
  schannel: next InitializeSecurityContext failed: SEC_E_ILLEGAL_MESSAGE (0x80090326) - This error usually occurs when a fatal SSL/TLS alert is received (e.g. handshake failed).
```

I ignored this since - and that's important here perhaps - I'm using CI to build the site and NEWS were build correctly on CI. But because I'm using CI, I can't check the problem with ToC on GitHub, right? I mean - even that I'm using dev version of `pkgdown`, I'm using this locally and CI uses other version?

To summarize: locally, when using `pkgdown::build_articles()` toc is visible, but it's not visible after building the site using CI on GitHub: https://gsmolinski.github.io/shinybreakpoint/articles/app-structure.html, https://pkgdown.r-lib.org/articles/customise.html. I have also removed the cookies by `Ctrl + F5` and also opened the site on the browser I didn't use for this before and still no ToC.

I think `pkgdown` on CI is not something I should / can update? I mean, I basically don't have almost any experience with CI, but if the problem is with CI and I can just wait when `pkgdown` will be updated on the server which CI uses, then I can just wait and this issue can be closed :)

**Comment by maelle** [comment]
You could add a dependency on dev pkgdown to your DESCRIPTION file for the time being, `usethis::use_dev_package("pkgdown")`, so CI would install it, but that field should be removed from DESCRIPTION before you submit your package to CRAN. 

We'll do a release "soon-ish" so you can also just wait.

Regarding the other error, here is a workaround 

> A workaround that probably works is to set the CURL_SSL_BACKEND="openssl" env var, before starting R. https://github.com/r-lib/pkgdown/issues/2211#issuecomment-1281171018

**Comment by gsmolinski** [comment]
Thank you! I will wait then. Thank you very much for `CURL_SSL_BACKEND` as well. I'm closing this since for me everything is clear.
